### PR TITLE
adding `content_for(:javascript)`

### PR DESCRIPTION
**Pull request by ghost**
This fixes an issue with https://github.com/diaspora/diaspora/pull/7194. E.g. JS errors on `/help`.

### CODE REVIEW

**Comment by svbergerem** [comment]
We have some randomly failing tests but you really broke one spec: https://travis-ci.org/diaspora/diaspora/jobs/177520922#L692

The help view is not working on the mobile view and with your PR it just shows a blank page. I don't know why you changed the filename of app/views/help/faq.html.haml to app/views/help/faq.haml (making it usable for the mobile view although it doesn't show anything there) but I'd just remove that change from this PR. (unless there are good reasons to keep it)

**Comment by jramos** [comment]
Whoops. That was a change intended for a different branch. I've removed it.

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -5,19 +5,21 @@
...
+      <!--[if IE]>
+      = javascript_include_tag :ie
+      <![endif]-->
 
-    <!--[if IE]>
-    = javascript_include_tag :ie
-    <![endif]-->
+      = javascript_include_tag :main, :templates
+      = load_javascript_locales
+      = translation_missing_warnings
```
Comment (svbergerem) [inline app/views/layouts/application.html.haml]: This line adds some CSS (https://github.com/diaspora/diaspora/blob/365441ede260212c406bc5ee4bde2764c1ff5e36/app/helpers/layout_helper.rb#L39), so I wouldn't add this to `content_for :javascript` but move it next to the color themes.

**Review by svbergerem — CHANGES_REQUESTED** [review]
One minor remark. Everything else looks good to me.

**Review by svbergerem — APPROVED** [review]

**Comment by svbergerem** [comment]
Merged as 2ca42ea5e6deb624e8ba9eea6b2af19f2b5ae672. Thanks!

**Pull request closed without merging**